js/package.json

```diff
@@ -29,6 +29,8 @@
     "build:app": "webpack --config webpack/app",
     "build:lib": "webpack --config webpack/libraries",
     "build:dll": "webpack --config webpack/vendor",
+    "build:markdown": "babel-node ./src/jsonrpc/generator/build-markdown.js",
```

Comment:
Probably not for this PR, but we should probably move the `build-*.js` out into (maybe?) `js/scripts` - it ended up being in there for no reason apart from the fact that we didn't want to lose code while combining the repos. (And as you've noticed was quite outdated and not being run)

Since it is building and not really source, it is a bit misleading atm.